packages/openactive-integration-tests/test/features/restrictions/booking-window/implemented/opportunity-outside-range-c1-c2-test.js

```diff
@@ -0,0 +1,82 @@
+const chakram = require('chakram');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { GetMatch, C1, C2 } = require('../../../../shared-behaviours');
+
+/**
+ * @typedef {import('chakram').ChakramResponse} ChakramResponse
+ */
+
+/**
+ * @param {C1|C2} stage
+ * @param {() => ChakramResponse} responseAccessor This is wrapped in a
+ *   function because the actual response won't be available until the
+ *   asynchronous before() block has completed.
+ */
+function itShouldReturn409Conflict(stage, responseAccessor) {
+  it('should return 409', () => {
+    stage.expectResponseReceived();
+    chakram.expect(responseAccessor()).to.have.status(409);
+  });
+}
+
+/**
+ * @param {C1|C2} stage
+ * @param {() => ChakramResponse} responseAccessor This is wrapped in a
+ *   function because the actual response won't be available until the
+ *   asynchronous before() block has completed.
+ */
+function itShouldReturnOpportunityOfferPairNotBookableError(stage, responseAccessor) {
+  it('should return OpportunityOfferPairNotBookableError', () => {
+    stage.expectResponseReceived();
+
+    const response = responseAccessor();
+    const errors = response.body.orderedItem.map(oi => oi.error && oi.error[0] && oi.error[0]['@type']).filter(e => !!e);
+
+    chakram.expect(errors.every(e => e === 'OpportunityOfferPairNotBookableError')).to.equal(true);
+  });
+}
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'restriction',
```

Comment:
```suggestion
  testCategory: 'restrictions',
```

Category doesn't match file path (so rendering the [mapping](https://github.com/openactive/openactive-test-suite/blob/master/packages/openactive-integration-tests/test/helpers/mapping.js) will fail)